pkgs/applications/office/agenda/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, pantheon
+, cmake
+, pkg-config
+, vala
+, gettext
+, glib
+, gtk3
+, libgee
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "agenda";
+  version = "1.0.12";
+
+  src = fetchFromGitHub {
+    owner = "dahenson";
+    repo = "agenda";
+    rev = version;
+    sha256 = "128c9p2jkc90imlq25xg5alqlam8q4i3gd5p1kcggf7s4amv8l8w";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gettext
+    vala
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    libgee
+    pantheon.granite
+  ];
+
+  passthru = {
+    updateScript = pantheon.updateScript {
+      attrPath = pname;
+    };
+  };
+
+  meta = with stdenv.lib; {
+    description = "A simple, fast, no-nonsense to-do (task) list for elementary OS";
```

```suggestion
    description = "A simple, fast, no-nonsense to-do (task) list designed for elementary OS";
```

I believe a lot of the descriptions have that `designed for elementary OS`, so it would help with `nix search`

---

You're totally right :+1: